packages/SwingSet/src/kernel/kernel.js

```diff
@@ -1178,7 +1182,7 @@ export default function buildKernel(
   }
 
   async function processDeliveryMessage(message) {
-    kdebug(`processQ ${JSON.stringify(message)}`);
+    kdebug(`\nprocessQ ${JSON.stringify(message)}`);
```

Comment:
Oops.  That was just a little bit of debug hackery that I missed removing (I was finding myself editing the verbose log after every run to put a line break between cranks because it was easier to read that way).  But making that format change official has much to recommend it, so if you'd prefer an extra `kdebug` call over an explicit newline char I'm OK with that.